spec/views/form_errors_builder_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/partners/_form.html.erb", type: :view do
+
+  let(:provinces) { Province.all }
```

Comment:
Any advantage to splitting up the setup into `let` & `before` instead of
```ruby
before :each do
  assign :provinces, Province.all
  assign :statuses, Status.all
  partner = build :partner, name: nil, region: 'Test'
  assign :partner, partner
  ..
end
```
(syntax differences not meant to be suggestions :smile: )